scalafmt-cli/src/main/scala/org/scalafmt/cli/ScalafmtCoreRunner.scala

```diff
@@ -38,7 +38,8 @@ object ScalafmtCoreRunner extends ScalafmtRunner {
         val termDisplay =
           newTermDisplay(options, inputMethods, termDisplayMessage)
         val exitCode = new AtomicReference(ExitCode.Ok)
-        inputMethods.par.foreach { inputMethod =>
+        // TODO add parallel collections
```

Comment:
You can use the `libraryDependencies ++= scalaBinaryVersion.value match { ... }` trick to fix this problem

https://mvnrepository.com/artifact/org.scala-lang.modules/scala-parallel-collections

Reply:
Parallel collections in 2.13 requires
```
import scala.collection.parallel.CollectionConverters._
```
which can't compile in 2.12 and 2.11

There is a [hack](https://github.com/scala/scala-parallel-collections/issues/22) for it but do we really need a parallel collection here?

Reply:
The general trick to work around this is to introduce a `Compat` object in `scala-2.13`, `scala-2.12`, ..., source directories 

```scala
// scala 2.13
object Compat extends ... {
  // ...
}
// scala 2.12
object Compat {
  // empty
}
```

Reply:
This trick is BTW not a "hack" in my book, it's a pretty standard solution to deal with issues like this. A general principle to follow when upgrading versions is to preserve the existing behavior as much as possible. I think we should keep parallel collections since they provide great speedups for large workspaces.